pkg/hookd/config/config.go

```diff
@@ -32,8 +32,8 @@ type Config struct {
 	LogLinkFormatter       string        `json:"log-link-formatter"`
 	MetricsPath            string        `json:"metrics-path"`
 	ProvisionKey           string        `json:"provision-key"`
-	TeamsAPIKey            string        `json:"teams-api-key"`
-	TeamsURL               string        `json:"teams-url"`
+	NaisAPITarget          string        `json:"nais-api-target"`
+	NaisAPIInsecure        bool          `json:"nais-api-insecure"`
```

Comment:
What does "insecure" mean?
I don't mind having "insecure" in the name, but I'd prefer if the option reflected what it actually does.